File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/atendimento/atendimento/dto/AtendimentoProcedimentoDTO.java
package br.com.grupopipa.gestaointegrada.atendimento.atendimento.dto;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class AtendimentoProcedimentoDTO {

    private UUID id;
    private UUID procedimentoId;
    private String procedimentoCodigo;
    private String procedimentoDescricao;
    private UUID tabelaItemId;
    private BigDecimal tabelaItemValor;
    private LocalDateTime dataInicio;
    private LocalDateTime dataFim;
}

File: src/frontend/src/app/components/atendimento/atendimento/model/atendimento-procedimento-dto.ts
import { Exclude, Expose } from 'class-transformer';

@Exclude()
export class AtendimentoProcedimentoDTO {
  @Expose() id?: string;
  @Expose() procedimentoId?: string;
  @Expose() procedimentoCodigo?: string;
  @Expose() procedimentoDescricao?: string;
  @Expose() convenioId?: string;
  @Expose() convenioNome?: string;
  @Expose() tabelaItemId?: string;
  @Expose() tabelaItemValor?: number;
  @Expose() dataInicio?: string;
  @Expose() dataFim?: string;
}
